Chrome/bridge.js

```diff
@@ -10,7 +10,7 @@
 var bridge_error = false;
 var bridge_error_object;
 var xkit_storage = {};
-var bridge_ver = "2.1";
+var bridge_ver = "2.2.1";
```

Comment:
The version in the comment on line 4 was bumped in commit b308886 but `bridge_ver` wasn't bumped back then so I'm just making everything consistent.